### ISSUE TITLE
Enable tail on job's log output

### ISSUE THREAD

**Issue by nitdana** [rundeck/rundeck#1154]
Hi,

It would be great to have the ability to watch the current log output for a massive output job, instead of waiting for rundeck to scroll until the current state.
Thanks!

**Comment by johntbush** [comment]
yes please! I have long running jobs with a lot of output, the current way this works now is painful.

**Comment by ireuben** [comment]
Would be cool if you when scrolled *up* it loaded in more of the previous lines above, but I'd be fine if it just dropped you into the current end of the file and then added new lines below from there (like "tail -fn 50 log_file".

**Comment by teopost** [comment]
+1

**Comment by HowardHong** [comment]
+1

**Comment by acqant** [comment]
+1

**Comment by lancepowell** [comment]
+1

**Comment by AWKIF** [comment]
+1

**Comment by lpaolotti** [comment]
+1

**Comment by midokate** [comment]
+1

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
